src/pages/basic_data/BasicDataPage.tsx

```diff
@@ -104,22 +104,21 @@ const BasicDataPage = () => {
               defaultPath={'/basic-data/institutions'}
               dataTestId={dataTestId.basicData.institutionsAccordion}>
               <NavigationList>
-                <LinkButtonRow>
-                  <LinkButton
-                    data-testid={dataTestId.basicData.adminInstitutionsLink}
-                    isSelected={currentPath === UrlPathTemplate.BasicDataInstitutions && !newCustomerIsSelected}
-                    to={UrlPathTemplate.BasicDataInstitutions}>
-                    {t('common.institutions')}
-                  </LinkButton>
-                  <LinkIconButton
-                    data-testid={dataTestId.basicData.addCustomerLink}
-                    isSelected={newCustomerIsSelected}
-                    to={getAdminInstitutionPath('new')}
-                    title={t('basic_data.institutions.add_institution')}
-                    icon={<AddBusinessIcon />}
-                  />
-                </LinkButtonRow>
+                <LinkButton
+                  data-testid={dataTestId.basicData.adminInstitutionsLink}
+                  isSelected={currentPath === UrlPathTemplate.BasicDataInstitutions && !newCustomerIsSelected}
```

Comment:
Ikke sikker men lurer på om denne kan fjernes nå?
```suggestion
                  isSelected={currentPath === UrlPathTemplate.BasicDataInstitutions}
```

Reply:
Den må være der med hvordan oppsettet på `Create`-knappen er akkurat nå, da den ikke er selected basert på path. Usikker på om jeg skal endre det eller beholde det slik det er nå.

Reply:
ah ja, stemmer. Litt grisete den der også menmen :)